ui/ducks/send/swap-and-send-utils.ts

```diff
@@ -208,3 +210,50 @@ export async function getSwapAndSendQuotes(request: Request): Promise<Quote[]> {
 
   return newQuotes;
 }
+
+const CHAIN_ID_TO_API_NAME: Record<string, string> = {
+  [CHAIN_IDS.MAINNET]: 'ethereum',
+  [CHAIN_IDS.BSC]: 'bsc',
+  [CHAIN_IDS.POLYGON]: 'polygon',
+  [CHAIN_IDS.AVALANCHE]: 'avalanche',
+  [CHAIN_IDS.ARBITRUM]: 'arbitrum',
+  [CHAIN_IDS.OPTIMISM]: 'optimism',
+  [CHAIN_IDS.ZKSYNC_ERA]: 'zksync',
+  [CHAIN_IDS.LINEA_MAINNET]: 'linea',
+  [CHAIN_IDS.BASE]: 'base',
+};
+
+export async function getDisabledSwapAndSendNetworksFromAPI(): Promise<
+  string[]
+> {
+  try {
+    const blockedChains: string[] = [];
+
+    const url = `${BASE_URL}/featureFlags`;
+
+    const featureFlagResponse = await fetchWithCache({
+      url,
+      fetchOptions: {
+        method: 'GET',
+        headers: { 'X-Client-Id': SWAPS_CLIENT_ID },
+      },
+      cacheOptions: { cacheRefreshTime: 0, timeout: MINUTE * 5 },
+      functionName: 'getDisabledSwapAndSendNetworks',
+    });
+
```

Comment:
Why not reuse `fetchSwapsFeatureFlags` here?

Reply:
Good call